docs/API.md

```diff
@@ -19,6 +19,43 @@ https://github.com/genome-nexus/genome-nexus-annotation-pipeline
 
 They are similar in terms of feature completeness
 
+## Annotation API
+### Send a variant
+- When a variant is sent to Genome Nexus, there are some other parameters sent in the query as well, such `isoformOverrideSource` and `fields`.
+- Check if there is an `isoformOverrideSource` provided in the query.
+  - If no isoform override source: use `uniprot` as the default.
```

Comment:
@leexgh we should prolly change this as we don't really maintain the `uniprot` isoform list anymore